src/components/green-flag/green-flag.css

```diff
@@ -3,6 +3,8 @@
     height: 1.1rem;
     opacity: 0.5;
     margin: 0.25rem 0.6rem;
+    user-drag: none; 
```

Comment:
Nope, my mistake. I tried a few different strategies and had that leftover. Removed